Packs/DomainToolsIrisDetect/Integrations/DomainToolsIrisDetect/README.md

```diff
@@ -0,0 +1,1124 @@
+   | Tags | Sets tags to be applied to indicators returned by this integration.                                                                                                                                                                                                                                                                                                                                                                           | False |
+   | Traffic Light Protocol Color | The Traffic Light Protocol \(TLP\) designation to apply to indicators fetched from the feed                                                                                                                                                                                                                                                                                                                                                   | False |
+   | Bypass exclusion list | When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.                                                                                                                                                                                                                                        | False |
+   | Incremental Feed | Incremental feeds pull only new or modified indicators that have been sent from the integration. As the determination if the indicator is new or modified happens on the 3rd-party vendor's side, and only indicators that are new or modified are sent to Cortex XSOAR, all indicators coming from these feeds are labeled new or modified.                                                                                                  | False |
+   | Incident type | Optionally specify an incident type for incidents created by this integration to work with specific playbooks                                                                                                                                                                                                                                                                                                                                 | False |
+   | Fetch incidents | This is a required field by XSOAR and should be set to 3, one for each possible feed type: new, changed, blocked.                                                                                                                                                                                                                                                                                                                             | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
```

Comment:
```suggestion
4. To ensure that fetch incidents works:
   1. Select the **Fetches incidents** radio button.
   2. Write here the rest of the required steps.
5. Click **Test** to validate the URLs, token, and connection.
```
Please add to the README an explanation on how to configure the fetch incident (what to choose in the classifier, etc.)